src/layers/tile/util.rs

```diff
@@ -10,18 +10,24 @@ pub(crate) fn parse_data_line(
     parser: &mut impl Iterator<Item = XmlEventResult>,
     tilesets: &[MapTilesetGid],
 ) -> Result<Vec<Option<LayerTileData>>, TiledError> {
-    match (encoding, compression) {
+    match (encoding.as_deref(), compression.as_deref()) {
         (None, None) => {
-            return Err(TiledError::Other(
-                "XML format is currently not supported".to_string(),
-            ))
+            return Err(TiledError::InvalidEncodingFormat {
```

Comment:
Here, the previous error was more useful. If there is no encoding specified, the original "XML" encoding should be assumed, where each tile gets its own `<tile />` element. Since this format is marked deprecated in Tiled, I would personally remove the "currently", since that indicates there would be plans to support it in the future.

I would suggest to tweak `fmt::Display for TiledError` to be a little more specific again in case neither is provided.

And wen can actually remove this special `(None, None)` case here, cause it is the same as the `_` below.